pipeline/terraform/modules/stack/locals.tf

```diff
@@ -17,7 +17,7 @@ locals {
   # The max number of connections allowed by the instance.
   # specified at /infrastructure/critical/rds_id_minter.tf
   base_rds_instances             = 1
-  id_minter_rds_max_connections  = (local.base_rds_instances + local.extra_rds_instances) * 45
+  id_minter_rds_max_connections  = local.base_rds_instances * 45
```

Comment:
These changes were required to get the terraform to plan successfully following the RDS upgrade. Should probably have been in another PR!

Reply:
Same goes for the deletion related to RDS below that related specifically to scaling during reindex.